appservice/src/createAppService/SiteHostingPlanStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+import { AppServicePlanListStep } from './AppServicePlanListStep';
+import { IAppServiceWizardContext } from './IAppServiceWizardContext';
+
+export class SiteHostingPlanStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
```

Comment:
Where is this going to be used?  I don't see it being added to the wizard anyway.

Reply:
Never mind, saw your other PR.